presto-common/src/test/java/com/facebook/presto/common/predicate/TestRange.java

```diff
@@ -281,4 +284,43 @@ public void testJsonSerialization()
         range = Range.lessThanOrEqual(DOUBLE, Double.MAX_VALUE);
         assertEquals(range, mapper.readValue(mapper.writeValueAsString(range), Range.class));
     }
+
+    public static final SqlFunctionProperties PROPERTIES = SqlFunctionProperties.builder().setTimeZoneKey(UTC_KEY).setLegacyTimestamp(true).setSessionStartTime(0).setSessionLocale(ENGLISH).setSessionUser("user").build();
```

Comment:
move this variable to top of the class; consider adding line breaks for readability, e.g. before .setTimeZoneKey, .setLegacyTimestamp, etc.

Reply:
done